src/WallGo/effectivePotential.py

```diff
@@ -51,14 +51,17 @@ class EffectivePotential(ABC):
     """
     
     fieldCount: int
-    """Number of background fields in your potential. Your concrete potential must set this to a nonzero positive integer."""
+    """
+    Number of background fields in your potential.
+    IMPORTANT: YOUR CONCRETE POTENTIAL MUST SET THIS TO A NONZERO POSITIVE INTEGER.
+    """
 
-    effectivePotentialError = 1e-8
-    """Typical relative accuracy at which the effective potential can be computed.
+    effectivePotentialError: float
+    """
+    Typical relative accuracy at which the effective potential can be computed.
     For simple polynomial potentials this is probably close to machine precision of Python floats (1e-15).
     For loop-corrected potentials a limited factor can be the eg. accuracy of numerical integration.
-    Default is 1e-8, matching the relative error in Jb/Jf thermal 1-loop integrals
-    from PotentialTools.
+    IMPORTANT: YOUR CONCRETE POTENTIAL MUST SET THIS TO A NONZERO POSITIVE FLOAT.
```

Comment:
Can we have a default value? Or is it too risky?

Reply:
I would avoid having a default value because it really depends on the potential implementation. With a default, it would be too easy for the user to just forget to change it.

Reply:
OK. This is still used only for finding a good temperature step size for derivatives, right? I feel this is something we need to improve in the future to make things easier for the user. But this is a future issue